blog/2015/03/first-software-carpentry-korea.html

```diff
@@ -0,0 +1,78 @@
+</p>
+<!-- end excerpt -->
+
+<p> Due to lack of instructors, Victor did teach all the lessons. Unfortunately, Dave, who is supposed to be a helper, could attend the first workshop such that one of attendees, Mr. Seo, became a volunteer and did a wonderful job. Even though there were some software installation problems, Mr. Seo and Victor could overcome some technical difficulties. </p>
+
+<p>Initially we want to start small, but we had some concerns about churn rates. Therefore, we initially limit the number of attendees up to 23. Because the first software carpentry workshop was free, the registration has been closed two weeks ago. The final attendees have been check by <a href="http://www.rapa.or.kr/">RAPA(host)</a> and the number was 17. Because the lesson contents of software carpentry are quite new, sort of advanced to Korean, and tailered to science and engineering, from the beginning Victor and RAPA paid special attentions to retention rates. Finally, we kept 12 attendees, which is over 70%. Our retention goal was 50%, 8 attendees.   
+</p>
+
+<p>
+<img src="software-carpentry-korea.jpg" alt="Software Carpentry Korea" />
+</p>
+
+<h2>CJK problems in software</h2>
+<p>
+We encountered a number of software difficulties, mostly CJK problems.
+</p>
+  <ul>
+    <li>
+    Nano editor: Korean characters are not recognized. Short-cut keys are not properly typed.
+    </li>
+    <li>
+    The software carpentry installers did not install all the software carpentry tools.
+    Therefore, manually check what has been installed and not installed, and then install not-installed ones.
+    </li>
+    <li>
+    Ipython notebook did not understand shell command "!", but DOS command works. We don't know why.
+    </li>
+  </ul>
+<h2>What we learned</h2>
+<p>
+The start of the first day we forgot gathering feedback (via sticky notes), but we explained how to use sticky notes and have got the feedback the end of the first day. Besides, we performed <a href="http://statkclee.github.io/xwmooc-sc/workshops/assess/pre-learner.html">pre-workshop questionnaire</a> and <a href="http://statkclee.github.io/xwmooc-sc/workshops/assess/post-learner.html">post-workshop questionnaire</a> through Google Survey, but survey responses are all Korean. A selection of the attendees' comments (random orders)are translated as follows:
+</p>
+<table class="table table-striped">
+ <thead>
+  <tr>
+   <th>Good points</th>
+   <th>Bad points</th>
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td>It was good to know various tools and concepts</td>
+   <td>When coding lively, typing was so fast!!! It's hard to catch up.</td>
+  </tr>
+  <tr>
+   <td>It was good to cover both theoretical and practical aspects of software based on software carpentry lessons and instructor's experiences</td>
```

Comment:
I followed the Korean dictionary ( http://endic.naver.com/enkrEntry.nhn?sLn=kr&entryId=1a0a09f4bfc54e6ba92cbcee225e5dcf ) ... maybe old...